packages/grafana-ui/src/components/MatchersUI/FieldTypeMatcherEditor.tsx

```diff
@@ -12,7 +12,7 @@ export const FieldTypeMatcherEditor = memo<MatcherUIProps<string>>(props => {
     (selection: SelectableValue<string>) => {
       return props.onChange(selection.value!);
     },
-    [counts, props.onChange]
+    [props]
```

Comment:
nit: Best practices is to destruct nested props and use them in the dependencies.

Reply:
done